.metadata/.log

```diff
@@ -0,0 +1,39 @@
+!SESSION 2018-08-11 16:55:20.540 -----------------------------------------------
```

Comment:
You need a proper `.gitignore`. Something like [gitignore.io](https://www.gitignore.io/) will help. The files are already in your remote, `git rm` and some cautious typing should help you out there.